app/controllers/api/usage_reports_controller.rb

```diff
@@ -0,0 +1,11 @@
+class API::UsageReportsController < ActionController::Base
+  def create
+    UsageReport.create!(params[:usage_report]) if ENV['HOSTED_BY_LOOMIO']
```

Comment:
This line makes me think that this wants to go into the loomio org plugin.

But actually there's a helper method in ApplicationController called `hosted_by_loomio?` which gets overridden (from 'false' to 'true' by the org plugin, so maybe that's the way to go? Just want to minimize the ways that we check for loomio.org.

```
UsageReport.create(params) if hosted_by_loomio?
```